### ISSUE TITLE
Module does not install the ops agent. Broken dependency (github cli)

### ISSUE THREAD

**Issue by fnfup** [terraform-google-modules/terraform-google-cloud-operations#102]
### TL;DR

Both this module and the script to install ops agent have a broken dependency when installing the github cli 

Error is the repo public key: `public key is not available: NO_PUBKEY 23F3D4EA75716059`

Script: curl -sSO https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh

### Expected behavior

Install ops agent on the target systems that have the appropriate labels and OS version specifications 

### Observed behavior

#012
rm: cannot remove '/etc/apt/sources.list.d/google-cloud-ops-agent.list': No such file or directory#012
Adding agent repository for ubuntu.
#012
deb https://packages.cloud.google.com/apt google-cloud-ops-agent-jammy-2 main#012
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).#012
OK#012
Reading package lists...#012
Building dependency tree...#012
Reading state information...#012
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.#012
W: Target Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-ops-agent.list:1 and /etc/apt/sources.list.d/google_osconfig_managed.list:3#012
W: Target Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-ops-agent.list:1 and /etc/apt/sources.list.d/google_osconfig_managed.list:3#012
W: Target Translations (main/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/google-cloud-ops-agent.list:1 and /etc/apt/sources.list.d/google_osconfig_managed.list:3#012
W: Target CNF (main/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/google-cloud-ops-agent.list:1 and /etc/apt/sources.list.d/google_osconfig_managed.list:3#012
W: Target CNF (main/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/google-cloud-ops-agent.list:1 and /etc/apt/sources.list.d/google_osconfig_managed.list:3#012
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease#012
Hit:2 http://us-east4.gce.archive.ubuntu.com/ubuntu jammy InRelease#012
Hit:3 http://us-east4.gce.archive.ubuntu.com/ubuntu jammy-updates InRelease#012
Hit:4 http://us-east4.gce.archive.ubuntu.com/ubuntu jammy-backports InRelease#012
Hit:5 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease#012
Hit:6 https://apt.releases.hashicorp.com jammy InRelease#012
Get:7 https://cli.github.com/packages stable InRelease [3917 B]#012
Hit:8 https://packages.cloud.google.com/apt google-cloud-ops-agent-jammy-2 InRelease#012
Hit:9 https://deb.nodesource.com/node_14.x jammy InRelease#012
Hit:10 https://packages.cloud.google.com/apt cloud-sdk InRelease#012
Err:7 https://cli.github.com/packages stable InRelease#012
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 23F3D4EA75716059#012
Reading package lists...#012

### Terraform Configuration

```hcl
module "compute_ops_agent_policy" {
  source  = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version = "0.4.0" # latest version see github repo above

  project_id = var.gcp_project_id
  policy_id  = "ops-agents-${var.deployment_prefix}-compute-policy"
  agent_rules = [{
    type               = "ops-agent"
    version            = "current-major"
    package_state      = "installed"
    enable_autoupgrade = true
  }]
  os_types = [{
    short_name = "ubuntu",
    version    = "22.04"
  }]
  group_labels = [{
    ops-agent = "enabled"
  }]
}
```

### Terraform Version

```sh
1.8.2
```
